content/90.back-matter.md

```diff
@@ -1,8 +1,19 @@
-{% if format is defined and format != "tex"-%}
-## References {.page_break_before}
+{% if format is not defined or format is defined and format != "tex"-%}
+### References <!-- $data-collapsed="true" --> {.page_break_before}
 
+<!-- 
+  This style exists in the accordion plugin already, but it will be loaded after 
```

Comment:
Is this a general issue with where the plugins are loaded, or is it a special case related to what we want to do with the collapsed section here?

Reply:
In general, I've tried to make the plugins such that they can work anywhere in the document, in case a manubot user changes that. This is a very specific instance of performance optimization; the `<style>` and `<div id="refs" data-collapsed="true">` would be done by the accordion plugin anyway because of the `### References <!-- $data-collapsed="true" -->`, they'd just be done a little bit further down the document.